Benchmark build failure: duplicate NFData instance for Proxy

```
/Users/danburton/scratch/union-0.1.1.0/bench/Benchmark.hs:11:10: error:
    Duplicate instance declarations:
      instance forall k (a :: k). NFData (Proxy a)
        -- Defined at bench/Benchmark.hs:11:10
      instance [safe] forall k (a :: k). NFData (Proxy a)
        -- Defined in ‘Control.DeepSeq’
```

How I reproduced the error:

```
stack unpack union-0.1.1.0
cd union-0.1.1.0
stack init --resolver nightly-2016-06-08
stack bench
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,12 @@
 sudo: false
 
+addons:
+  apt:
+    packages:
+      - libgmp-dev
+
 env:
- - GHCVER=7.10.3 STACK_YAML=stack.yaml
+ - STACK_YAML=stack.yaml
 
 cache:
   directories:
@@ -10,20 +15,14 @@ cache:
 before_install:
  - export PATH=/opt/ghc/$GHCVER/bin:$PATH
  - mkdir -p ~/.local/bin
- - travis_retry curl -L https://github.com/commercialhaskell/stack/releases/download/v1.0.2/stack-1.0.2-linux-x86_64.tar.gz | tar -xvz
- - mv stack-1.0.2-linux-x86_64/stack ~/.local/bin
+ - travis_retry curl -L https://github.com/commercialhaskell/stack/releases/download/v1.1.2/stack-1.1.2-linux-x86_64.tar.gz | tar -xvz
+ - mv stack-1.1.2-linux-x86_64/stack ~/.local/bin
  - export PATH=~/.local/bin:$PATH
  - stack --version
+ - stack setup
 
 install:
  - stack install --only-dependencies --no-terminal
 
 script:
  - stack build --test --bench --no-terminal --ghc-options -Werror
-
-addons:
-  apt:
-    sources:
-    - hvr-ghc
-    packages:
-    - ghc-7.10.3

--- a/bench/Benchmark.hs
+++ b/bench/Benchmark.hs
@@ -2,14 +2,10 @@
 {-# LANGUAGE PolyKinds #-}
 module Main (main) where
 
-import Control.DeepSeq
 import Control.Lens
 import Criterion.Main
 import Data.Dynamic
 import Data.Union
-
-instance NFData (Proxy (a :: k)) where
-  rnf Proxy = ()
 
 union1 :: OpenUnion '[(), Proxy 0, Proxy 1]
 union1 = openUnion # ()

--- a/stack.yaml
+++ b/stack.yaml
@@ -1,2 +1,2 @@
 packages: ['.']
-resolver: lts-5.1
+resolver: nightly-2016-06-12

--- a/union.cabal
+++ b/union.cabal
@@ -52,7 +52,7 @@ benchmark bench
   build-depends:       base
                ,       union
                ,       lens
-               ,       deepseq
+               ,       deepseq >=1.4.2
                ,       criterion
   hs-source-dirs:      bench
   default-language:    Haskell2010

--- a/union.cabal
+++ b/union.cabal
@@ -1,5 +1,5 @@
 name:                union
-version:             0.1.1.0
+version:             0.1.1.1
 synopsis:            Extensible type-safe unions
 description:
 

